src/lib/ui/SupportedDevices/SupportedDevices.tsx

```diff
@@ -0,0 +1,17 @@
+import SupportedDeviceRow from './SupportedDeviceRow.js'
+import SupportedDevicesHeader from './SupportedDevicesHeader.js'
+
+export default function SupportedDevices() {
+  return (
+    <table className='seam-supported-devices-table'>
```

Comment:
Using a native `table` element (as well as native `thead`, `th`, `tbody`, `tr`, and `td`, etc.) so that I don't have to do a lot of `!important` rewrites of the current `Table` component. The native browser styling works perfectly for this component. In addition, it avoids a lot of excess imports, etc.